tests/scenarios/package.json

```diff
@@ -81,6 +82,8 @@
     "ember-source-latest": "npm:ember-source@latest",
     "ember-truth-helpers": "^3.0.0",
     "execa": "^5.1.1",
+    "node-fetch": "2.7.0",
+    "strip-ansi": "^6.0.0",
```

Comment:
I had to use `strip-ansi@6.0.0` because `> 7.x` is an ESM-only package. Importing this into the scenarios files triggers an `Unknown file extension ".ts"` error or an `ERR_REQUIRE_ESM` error.